03-first-aws-environment/components/terraform/tfstate-backend/README.md

```diff
@@ -0,0 +1,94 @@
+# Component: `tfstate-backend`
+
+This component is responsible for provisioning an S3 Bucket and DynamoDB table that follow security best practices for usage as a Terraform backend.
+
+## Usage
+
+**Stack Level**: Regional
+
+Here's an example snippet for how to use this component.
+
+```yaml
+components:
+  terraform:
+    tfstate-backend:
+      vars:
+        name: tfstate
```

Comment:
we can also set attributes if we need a unique name.

Reply:
Yeah good point. I can update, but let's go full circle on the conversation above regarding if we want to keep `uniquify` or not.